Dockerfile

```diff
@@ -11,5 +11,9 @@ COPY requirements.txt /tmp/requirements.txt
 COPY ./bin/peep.py /tmp/peep.py
 RUN /tmp/peep.py install -r /tmp/requirements.txt
 
+# Symlink node because Ubuntu's node binary is named nodejs.
+RUN ln -s /usr/bin/nodejs /usr/bin/node
```

Comment:
Another way to do this is `apt-get install node-legacy`. Both work equally well, though I imagine an Ubuntu person would tell you the package is more correct.